Dockerfile.windows

```
@@ -1,57 +1,96 @@
-
-RUN setx /M Path "c:\ProgramData\chocolatey\bin;c:\Program Files\Git\usr\bin;%Path%;c:\gcc\bin"
+# Chocolatey and install curl/git
+RUN powershell -command sleep 5 ; iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))
+RUN powershell -command sleep 5; choco install curl -y -v
+RUN powershell -command sleep 5; choco install git -y -v --version=%GIT_VERSION% -params '"/GitAndUnixToolsOnPath"'
 
-RUN powershell -command \
-    sleep 2; \
-    curl.exe -L -k -o go.msi https://storage.googleapis.com/golang/go%GO_VERSION%.windows-amd64.msi; \
-    curl.exe -L -k -o gcc.zip http://downloads.sourceforge.net/project/tdm-gcc/TDM-GCC%205%20series/5.1.0-tdm64-1/gcc-5.1.0-tdm64-1-core.zip ; \
-    curl.exe -L -k -o runtime.zip http://downloads.sourceforge.net/project/tdm-gcc/MinGW-w64%20runtime/GCC%205%20series/mingw64runtime-v4-git20150618-gcc5-tdm64-1.zip ; \
-    curl.exe -L -k -o binutils.zip http://downloads.sourceforge.net/project/tdm-gcc/GNU%20binutils/binutils-2.25-tdm64-1.zip
+# Path
+RUN setx /M Path "c:\ProgramData\chocolatey\bin;c:\Program Files\Git\usr\bin;%Path%;c:\gcc\bin;c:\7zip"
 
-RUN powershell -command \
-    .\go.msi /quiet ; \
-    Expand-Archive gcc.zip \gcc -Force ; \
-    Expand-Archive runtime.zip \gcc -Force ; \
-    Expand-Archive binutils.zip \gcc -Force
+# Download everything else we need to install
+# We want a 64-bit make.exe, not 16 or 32-bit. This was hard to find, so documenting the links
+#  - http://sourceforge.net/p/mingw-w64/wiki2/Make/ -->
+#  - http://sourceforge.net/projects/mingw-w64/files/External%20binary%20packages%20%28Win64%20hosted%29/ -->
+#  - http://sourceforge.net/projects/mingw-w64/files/External binary packages %28Win64 hosted%29/make/
+RUN powershell -command sleep 5; curl.exe -L -k -o make.zip http://downloads.sourceforge.net/project/mingw-w64/External%20binary%20packages%20%28Win64%20hosted%29/make/make-3.82.90-20111115.zip
```

Just looked it up. I really don't think we need any of these options apart from L. Why fail silently (f) or be silent (s)??? I already know S is un-necessary when the curl does fail - you get a great error message, at least with this ported version ;) And I already have L - that is necessary for stupid sourceforge links!